charts/kubermatic/static/cleanup-container.yaml

```diff
@@ -29,10 +29,10 @@ env:
 - name: ACCESS_KEY_ID
   valueFrom:
     secretKeyRef:
-      name: s3-credentials
```

Comment:
should we change this for the Cleanup container? Its used for the old backups which use Minio so we should not touch it. The new ones use the Delete container.

Similar goes for the old store container

Reply:
I can revert this in the chart

Reply:
also in the operator resources

Reply:
yep, done already